meta-balena-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher

```diff
@@ -395,8 +379,10 @@ if [ "$LUKS" = "1" ]; then
     done
 
     # Store files necessary for TPM decryption to the EFI partitions
-    mv "$PERSISTENT_KEY_HANDLE" "$EFI_MOUNT_DIR/balena-luks.ctx"
-    mv "$ENCRYPTED_PASSPHRASE" "$EFI_MOUNT_DIR/balena-luks.enc"
+    mv "$TPM_RESULT_DIR/persistent.ctx" "$EFI_MOUNT_DIR/balena-luks.ctx"
+    mv "$TPM_RESULT_DIR/passphrase.enc" "$EFI_MOUNT_DIR/balena-luks.enc"
+
+    rm -rf "$TPM_RESULT_DIR"
```

Comment:
```suggestion
    rm -rf "$TPM_RESULT_DIR" && sync
```

Reply:
These are temporary files, currently used in two places, both in-memory only - initrd, which is in-memory by default and flasher where `$TPM_RESULT_DIR` is tmpfs, I don't think sync would help anything.